api/print.php

```diff
@@ -74,27 +82,52 @@
                 $url = $config['qr']['url'];
             }
             include '../vendor/phpqrcode/lib/full/qrlib.php';
-            QRcode::png($url, $filename_codes, QR_ECLEVEL_H, 10);
+            QRcode::png($url, $filename_codes, QR_ECLEVEL_H, $config['print']['qrSize']);
```

Comment:
Make the ECC adjustable too?

Reply:
Depends on how much impact it has. I didn't play around with it by now.

Reply:
Not much, another switch case would need to be added. There's 4 ECC options we could use. Plus the config stuff for the Adminpanel.

Reply:
In this case I'd stay away for it. But maybe choose a different level anyway? https://www.qrcode.com/en/about/error_correction.html
> Level Q or H may be selected for factory environment where QR Code get dirty
I don't feel like this is the typical use case or requirement for a photobooth...
https://blog.qrstuff.com/2011/12/14/qr-code-error-correction
> Level L or Level M represent the best compromise between density and ruggedness for general marketing use.

Reply:
There might be different use cases which why I'd consider to add a switch and let the user decide, but sure we can get down to a lower profile by default while on it.

Reply:
Might be enough to set $config['print']['qrEcc'] and make it a select list inside Adminpanel, so no need for a switch case

Reply:
Or maybe in general for QR and move this setting to QR section. Same applies to the qr inside gallery